src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,70 @@
+.stopwatch {
+  position: relative;
+  width: $width;
+  height: $width;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: calc(50% - #{$hand-width-minutes} / 2);
+    transform: translateX(-50%);
+    transform-origin: bottom center;
+    animation: spin-minute 3600s steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: $seconds-hand-height;
+    width: $hand-width-seconds;
+    background-color: $seconds-hand-color;
+    bottom: 50%;
+    left: calc(50% - #{$hand-width-seconds} / 2);
```

Comment:
Similar to the `.stopwatch__minutes-hand`, the left property for the `.stopwatch__seconds-hand` should also have its division operation wrapped in parentheses: `calc(50% - (#{$hand-width-seconds} / 2))`.